docs/topics/arrays.md

```diff
@@ -1,28 +1,47 @@
 [//]: # (title: Arrays)
```

Comment:
I've updated the page with a note at the beginning trying to reflect Roman's words. Instead of highlighting the negatives of arrays I've highlighted some benefits of collections that arrays don't have. Is this worded strongly enough in your view?